package.json

```diff
@@ -119,19 +112,13 @@
 		"prestart": "npx check-node-version --package && node bin/welcome.js",
 		"start": "yarn run -s build",
 		"poststart": "run-p -s start-build-if-web start-build-if-desktop",
-		"start-fallback": "DEV_TARGET=fallback yarn run start",
 		"start-jetpack-cloud": "CALYPSO_ENV=jetpack-cloud-development yarn start",
 		"start-jetpack-cloud-p": "PORT=3001 CALYPSO_ENV=jetpack-cloud-development yarn run build-server",
-		"poststart-jetpack-cloud-p": "PORT=3001 CALYPSO_ENV=jetpack-cloud-development yarn run start-build-web",
+		"poststart-jetpack-cloud-p": "PORT=3001 CALYPSO_ENV=jetpack-cloud-development yarn run start-build",
 		"reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,json,ts,tsx}\"",
-		"start-build-do": "node build/server.js | bunyan -o short",
-		"start-build-fallback": "BROWSERSLIST_ENV=defaults yarn run start-build-do",
-		"start-build-evergreen": "BROWSERSLIST_ENV=evergreen yarn run start-build-do",
-		"start-build-if-desktop": "node -e \"(process.env.CALYPSO_ENV === 'desktop' || process.env.CALYPSO_ENV === 'desktop-development') && process.exit(1)\" || yarn run start-build-fallback",
-		"start-build-if-web": "node -e \"(process.env.CALYPSO_ENV !== 'desktop' && process.env.CALYPSO_ENV !== 'desktop-development') && process.exit(1)\" || yarn run start-build-web",
-		"start-build-web": "run-p -s start-build-web-if-fallback start-build-web-if-evergreen",
-		"start-build-web-if-fallback": "node -e \"process.env.DEV_TARGET === 'fallback' && process.exit(1)\" || yarn run start-build-fallback",
-		"start-build-web-if-evergreen": "node -e \"(! process.env.DEV_TARGET || process.env.DEV_TARGET === 'evergreen') && process.exit(1)\" || yarn run start-build-evergreen",
+		"start-build": "node build/server.js | bunyan -o short",
```

Comment:
Shouldn't this have `BROWSERSLIST_ENV=calypso`? Or does the server environment no longer need to know which `browserslist` target the client side is running?

Reply:
Good catch! I think it should have the env var, yes.